### ISSUE TITLE
BUG Using browser-sdk with NextJS 12

### ISSUE THREAD

**Issue by puhoshville** [whereby/browser-sdk#51]
```
Error: Failed to fetch update manifest Internal Server Error
    at http://localhost:3000/_next/static/chunks/fallback/webpack.js?ts=1686740013295:1176:37
ReferenceError: exports is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and '/bla/bla/bla/node_modules/@whereby.com/browser-sdk/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
```
Hello! Failed to use library (`2.0.0-alpha9` version)  with NextJS. Seems like it can be fixed by removing `"type": "module"` from package.json.

Please assist how to overcome this issue to use with NextJS 12.0?

**Comment by havardholvik** [comment]
Hi there 👋 We will try to replicate this issue and get back to you shortly with instructions on how to proceed. Might be that we have misconfigured our package, which might require us to provide a new version for you to test.

**Comment by puhoshville** [comment]
@havardholvik thank you for really fast reply :) yeap, I would be happy if you will be able to replicate the issue and release new version! if I can help somehow – pls tag me here, will try to do my best!)

**Comment by puhoshville** [comment]
@havardholvik  Hello!
Do you have an idea when this release will be available for end users?
Trying to plan our product development tasks.

Or maybe any idea how I can fix it on my own? I tried to rename all `.js` files into `.cjs` in https://github.com/puhoshville/browser-sdk/tree/development
But after installing it using `yarn add https://github.com/puhoshville/browser-sdk.git#development` I'm getting error like: 
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'path\to\project\frontend\node_modules\@whereby.com\browser-sdk\' 
```
(both for windows and mac)

**Comment by DTX-92** [comment]
Same probkem for me. 
![image](https://github.com/whereby/browser-sdk/assets/20479161/bc3787ac-de22-4a45-adb2-c384564520f0)
Cant start it running with any react thourh any technologies
![image](https://github.com/whereby/browser-sdk/assets/20479161/d555fae3-b9fe-4279-92cf-51c61d52137a)

any example with fresh packet and react possible?

**Comment by havardholvik** [comment]
> Same probkem for me. ![image](https://user-images.githubusercontent.com/20479161/246638278-bc3787ac-de22-4a45-adb2-c384564520f0.png) Cant start it running with any react thourh any technologies ![image](https://user-images.githubusercontent.com/20479161/246638270-d555fae3-b9fe-4279-92cf-51c61d52137a.png)
> 
> any example with fresh packet and react possible?

@DTX-92; I dont think this is the same issue, as from your stack trace you seem to be using our ESM build. 

Looking closer at your error, it seems this might be related to using our SDK server-side, as is the default mode for Next. Have you looked at https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading#with-no-ssr to configure your custom video component to only render client side?

**Comment by havardholvik** [comment]
> Using the most recent next version, I dont see this issue. However, I am able to reproduce your issues now @puhoshville , when creating a vanilla app using `npx create-next-app@12`.
> 
> So it seems we might have some inconsistencies in our package.json, mainly having `type: "module"` and a `main` field pointing to our CJS build. However, we also have a `module` field pointing to our ESM build, which seems to be respected by the bundlers I have used up until this point.
> 
> One quick fix we can do is to update the CJS filename from `.cjs.js` -> `.cjs` and leave the `type` field as is. According to https://nodejs.org/docs/latest-v13.x/api/esm.html#esm_package_json_type_field, any compiler should treat .cjs as a common-js module regardless of the value in the type field.
> 
> Will let you know when we have issued a new version 👍

@puhoshville : We have now released `2.0.0-alpha10`. I've tested that with your version of Next and things seem to be working fine now out of the box. Let me know if this solves the issue for you. 

Keep in mind though that you need to ensure any component which makes use of the SDK is rendered client side, as we are using apis only available in the browser.
